packages/plugins/geth/src/index.js

```diff
@@ -23,6 +23,11 @@ class Geth {
     }
 
     this.events.request("blockchain:node:register", constants.blockchain.clients.geth, {
+      isStartedFn: (isStartedCb) => {
+        this._doCheck((state) => {
+          return isStartedCb(null, state.status === "on");
```

Comment:
Just a general comment: it would be great if `state` was typed and `state.status` was an enum that represents all possible states of whatever `state` can be.

Reply:
Another tiny naming suggestion, but not a must: how about we rename `isStartedCb` to `done`?

Reply:
`isStarted` is meant to represent a function that checks whether or not the blockchain is already started. I'm not sure `done` would describe this properly? I agree that if we moved this to TypeScript and had everything concretely typed, naming wouldn't be as much of an issue as we could get some help from the typechecker.

Reply:
>  I'm not sure done would describe this properly?

Good point. This callback gets called multiple times over time.